packages/brick_offline_first/lib/src/offline_queue/offline_queue_graphql_client.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:brick_offline_first/src/offline_queue/graphql_request_sqlite_cache_manager.dart';
+import 'package:brick_offline_first/src/offline_queue/graphql_request_sqlite_cache.dart';
+import 'package:gql_link/gql_link.dart';
+import 'package:gql_exec/gql_exec.dart';
+import 'package:logging/logging.dart';
+
+/// Stores all requests in a SQLite database
+class OfflineQueueGraphqlClient extends Link {
```

Comment:
With the file being similarly renamed
```suggestion
class GraphqlOfflineQueueLink extends Link {
```